ckanext/imageview/tests/test_view.py

```diff
@@ -11,21 +11,20 @@
 class TestImageView(helpers.FunctionalTestBase):
 
     @classmethod
-    def setup_class(cls):
+    def _apply_config_changes(cls, cfg):
+        cfg['ckan.plugins'] = 'image_view'
```

Comment:
better yet, use `_load_plugins = ['image_view']` at the class level and FunctionalTestBase will take care of loading and unloading the plugins

Reply:
i've done this, because it was complaining with `template not found`  error

Reply:
i see what have you meant, whole day debugging got me :)